src/test/java/tools/jackson/databind/deser/bean/BeanDeserializerTest.java

```diff
@@ -498,4 +517,24 @@ public void testAddOrReplacePropertyIsUsedOnDeserialization() throws Exception {
         Issue1912Bean result = mapper.readValue("{\"subBean\": {\"a\":\"foo\"}}", Issue1912Bean.class);
         assertEquals("foo_custom", result.subBean.a);
     }
+
+    // https://github.com/FasterXML/jackson-databind/issues/5008
+    @Test
+    public void testSimpleValue5008() throws Exception {
+        // according to #5008, this only started working in v2.18.0
+        SimpleValue5008 value = MAPPER.readValue(
+                a2q("{'value':'abc123'}"), SimpleValue5008.class);
+        assertEquals("abc123", value.value);
+    }
+
+    // https://github.com/FasterXML/jackson-databind/issues/5008
+    @JacksonTestFailureExpected
```

Comment:
Not valid: only one of these behaviors can be supported at given time -- it's either delegating (String) or properties-based (Object). Jackson does not support using 2 different shapes for a single Creator method.

Reply:
Clarification: what I mean is that `@JacksonTestFailureExpected` really is only meant to be used for things where we eventually expect (or hope, at least) to change behavior. I don't think there should be tests that will always be expected to fail.